contracts/round/RoundImplementation.sol

```diff
@@ -128,9 +128,6 @@ contract RoundImplementation is AccessControlEnumerable, Initializable {
   /// @notice MetaPtr to the application form schema
   MetaPtr public applicationMetaPtr;
 
-  /// @notice MetaPtr to the projects
-  MetaPtr public projectsMetaPtr;
```

Comment:
since this is an upgradeable contract, we can't remove variables that change the storage of the proxy

Reply:
we actually don't use the contract itself, just call one function and the clone library just takes the address and loads the bytecode directly from the roundImplementation address

Reply:
awesome you are right @KurtMerbeth ! so as long as we don't upgrade old rounds this is totally fine! thank you 🚀